### ISSUE TITLE
Various compats curios tag mismatched or missing.

### ISSUE THREAD

**Issue by ThePiekar** [userofbricks/Expanded-Combat#61]
Hi, 
I'm sorry for posting such a seemingly broad issue, let me be concrete.
I have figured out the issue on my end with datapacks, so I know exactly what's wrong.

The base mod uses "quiver_ec" for curio quiver tag, while the expansions use just "quiver".
Making these not work (surprisingly MEI suggests they are still tagged as quivers, just no quiver_ecs).
Once I applied my datapack (list of all misbehaving quivers in quiver_ec.json in an appropriate folder structure), they all started reporting as just quiver tag, and working as intended.)

Additionally, gauntlets from specifically Alex's caves compat are not tagged.
(I am not using the aether, twilight forest, ars, apotheosis, blue skies, epic samurai and advanced netherite compats though, so I cannot comment on these)

I also couldn't help but notice, only one compat has a repository.
I of course respect any and all decisions about distributing one's work, but I'd be more than happy, if such a need arises, to make separate pull requests in the future, given such a repository would materialize.

TL:DR change "quiver.json" to "quiver_ec.json" in aquaculture, enderite, gilded netherite/enderite, create, cataclysm (possibly more).
fix alex's caves gauntlets json (possibly more)

**Comment by userofbricks** [comment]
The naming scheme for the tags changed in the last update for ec so literally every tag for arrows, quivers and gauntlets needs renaming. I forgot to start doing that earlier than this but I will get on it soon.

**Comment by userofbricks** [comment]
updates to all compats that have quivers have been uploaded.
